app/views/HasCommodityCodeView.scala.html

```diff
@@ -38,16 +38,21 @@
 
 @searchLink = {
     @link(
-            id = "search_for_commodity_code",
-            text = messages("hasCommodityCode.paragraph.1.link"),
-            call = Call("GET", config.findCommodityCodeUrl)
-        )
-    }
+        id = "search_for_commodity_code",
+        text = messages("hasCommodityCode.linkText"),
+        call = Call("GET", config.findCommodityCodeUrl)
+    )
+}
+
+@detailsContent = {
+    <p class="govuk-body">
+         @messages("hasCommodityCode.paragraph.1")
+    </p>
+    <p class="govuk-body">
+        @messages("hasCommodityCode.youCan") @searchLink @messages("hasCommodityCode.ifYouDontKnowIt")
+    </p>
+}
```

Comment:
there is a paragraph component in the service which can be used instead. 

`
     @paragraph(Html(messages("hasCommodityCode.paragraph.1")))

     @paragraph(Html(messages("hasCommodityCode.paragraph.2", searchLink)))
`